common/test/acceptance/pages/studio/overview.py

```diff
@@ -845,6 +845,15 @@ def deprecated_advance_modules(self):
         return self.q(css='.advance-modules-list li').text
 
 
+class CourseOutlineSignInRedirectPage(CourseOutlinePage):
+    """
+    This class may be used when user is not signed
+    in and accesses the course.
+    """
+    def is_browser_on_page(self):
+        return self.q(css='body.view-signin').present
```

Comment:
using `.visible` here will be stronger against flaky conditions. That is, assuming this element is visible.